### PR TITLE
feat: Created remember me option for passwords in login page

### LINKED ISSUE #1905: Remember me option for passwords in sign up page
**Describe the feature you'd like**

No " remember me "option in sign up page 

**Additional context**

Remember me checkbox option would help users remember their password and not have to type everytime.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #1905 

Changes:Done  a few additions in login activity class and its layout.

Screenshots for the change: 

fileupdate

### CODE REVIEW

**Comment by chanmol1999** [comment]
Please review it @iamareebjamal @batbrain7

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 1]
```diff
@@ -59,18 +64,33 @@ class LoginActivity : AppCompatActivity(), ILoginView {
...
         val string = bundle?.getString("email")
         if (string != null)
             email.editText?.setText(string)
     }
 
+    private fun remember() {
+        val editTextUsername = findViewById<EditText>(R.id.emailInput)
+        val editTextPassword = findViewById<EditText>(R.id.passwordInput)
+        val saveLoginCheckBox = findViewById<CheckBox>(R.id.remember)
+        val loginPreferences = getSharedPreferences("loginPrefs", MODE_PRIVATE)
```
Comment (batbrain7) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: no need to use this, kotlin can use the view elements directly

**Comment by chanmol1999** [comment]
Please review it now @batbrain7

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 2]
```diff
@@ -62,18 +61,28 @@ class LoginActivity : AppCompatActivity(), ILoginView {
...
             email.editText?.setText(string)
     }
 
+    private fun remember() {
+        val saveLoginCheckBox = findViewById<CheckBox>(R.id.remember)
+
+        if (getSharedPreferences("loginPrefs", MODE_PRIVATE).getBoolean("saveLogin", false) == true) {
+            email.editText?.setText(getSharedPreferences("loginPrefs", MODE_PRIVATE).getString("username", ""))
+            password.editText?.setText(getSharedPreferences("loginPrefs", MODE_PRIVATE).getString("password", ""))
+            saveLoginCheckBox.setChecked(true)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: Won't save the email and password in shared preferences. Highly insecure

Comment (chanmol1999) [reply app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: Can you please help me out @iamareebjamal  for a better option.

**Comment by iamareebjamal** [comment]
Read the discussion in #1301

**Comment by chanmol1999** [comment]
> Read the discussion in #1301

I did a study of credentials object and also read discussion but still something is missing out in this .Could you please help me out @iamareebjamal .Please have a look on my  last commit.

**Comment by iamareebjamal** [comment]
What is missing out?

**Comment by chanmol1999** [comment]
> What is missing out?

@iamareebjamal it is not working as required . Credentials of user do not come up on the login screen.

**Comment by iamareebjamal** [comment]
I've not worked with Google credentials management API so I don't know what may be wrong

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 3]
```diff
@@ -61,6 +69,20 @@ class LoginActivity : AppCompatActivity(), ILoginView {
...
 
+        credentialsClient = Credentials.getClient(this)
+        val hintRequest = HintRequest.Builder()
+                .setHintPickerConfig(CredentialPickerConfig.Builder()
+                        .setShowCancelButton(true)
+                        .build())
+                .setEmailAddressIdentifierSupported(true)
+                .build()
+
+        val intenthint = credentialsClient.getHintPickerIntent(hintRequest)
```
Comment (immadisairaj) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: variable name should be in camel case

File: app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt [hunk 4]
```diff
@@ -158,7 +196,14 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         val stringEmail = email.editText?.text.toString()
         val stringPassword = password.editText?.text.toString()
         val stringURL = inputUrl.editText?.text.toString()
-
+        val rem = findViewById<CheckBox>(R.id.remember)
```
Comment (immadisairaj) [inline app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt]: don't use short variable names

**Pull request closed without merging**